### PR TITLE
Kacybranch

### PR DESCRIPTION
**Pull request by imaginarynumbers**
Made some updates to the scrum team class. 
Had to add some getter methods in Employee class.

### CODE REVIEW

File: DatabaseObjects/ScrumMember.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    private int employeeId;
+    public Employee employee;
+
+    public ScrumMember(int scrumId, int employeeId) {
+        this.scrumId = scrumId;
+        this.employeeId = employeeId;
+    }
+    
+    public ScrumMember(Employee employee) {
+        this.employee = employee;
```
Comment (mouuff) [inline DatabaseObjects/ScrumMember.java:16]: so `this.employee` is only initialized when we call this constructor (and not when using `ScrumMember(int scrumId, int employeeId)`? why?

File: DatabaseObjects/ScrumTeam.java [hunk 2]
```diff
@@ -22,32 +23,41 @@ public ScrumTeam(ResultSet result) throws SQLException {
...
+        ResultSet keys = ps.getGeneratedKeys();
+        if (keys.next())
+            this.scrumId = keys.getInt(1);
+        else
+            System.err.println("Did not get generated key: " + query);
+    }
+    
+    public void insert(ScrumMember member, int scrumEmployeeId) {
+        ScrumMember newMember = new ScrumMember(0, scrumEmployeeId);
+        teamMembers.add(newMember);
```
Comment (mouuff) [inline DatabaseObjects/ScrumTeam.java:52]: so `teamMembers` is only contains members we inserted during execution?
(when we launch the program there will be a new instance of this class, so your array will be empty)

File: src/StateAddUserStory.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
+import java.sql.SQLException;
+
+public class StateAddUserStory extends State {
+    
+    private Project project;
+    
+    public StateAddUserStory(Project project) {
```
Comment (mouuff) [inline src/StateAddUserStory.java]: This code have been moved to StateAccessProject

**Review by mouuff — CHANGES_REQUESTED** [review]

**Comment by imaginarynumbers** [comment]
Changes need to be made, code not sufficient enough to merge into master.

**Pull request closed without merging**